### PR TITLE
return hard coded comments- PR 3

### PR DESCRIPTION
**Pull request by claraeverett**
Completed PR 3 with hard-coded comments.

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -23,6 +23,11 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <version>2.8.6</version>
+        </dependency>
```
Comment (sbahaud) [inline portfolio/pom.xml]: nit: Please fix the alignment - move </dependency> to the left.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -19,14 +19,31 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
```
Comment (sbahaud) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Order imports in alphabetical order.
com.google
java.util
javax.

File: portfolio/pom.xml (hunk 1 above)
Comment (claraeverett) [reply portfolio/pom.xml]: Done

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (claraeverett) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done

File: portfolio/pom.xml (hunk 1 above)
Comment (claraeverett) [reply portfolio/pom.xml]: Done

**Pull request merged**